### DISCUSSION TITLE
the SMV file cannot be viewed

### DISCUSSION THREAD

**Discussion by zzlcherry** [firemodels/fds#13734] [Q&A]
Hello! The question I want to ask is that although results are simulated using a supercomputer, the SMV file cannot be viewed. What's going on? Looking forward to your reply.

**Comment by zzlcherry** [comment]
![image](https://github.com/user-attachments/assets/554587df-0d14-43df-97fa-71cc02e94b59)

**Comment by mcgratta** [comment]
We cannot diagnose the problem with the information you have provided. I suggest that you

1. Make sure you are using the latest version of FDS and Smokeview
2. Run a simple case and if the problem persists, submit the case input file.

**Comment by zzlcherry** [comment]
Thank you for your answer.
